pallets/subtensor/src/macros/errors.rs

```diff
@@ -216,6 +216,8 @@ mod errors {
         ZeroMaxStakeAmount,
         /// Invalid netuid duplication
         SameNetuid,
+        /// Too frequent staking operations
+        StakingOperationRateLimitExceeded,
```

Comment:
It will shift the index of the error and mess with decoding. The error should be added at the end of the enum.